sdk/helper/custommetadata/custom_metadata.go

```diff
@@ -0,0 +1,66 @@
+package custommetadata
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-secure-stdlib/strutil"
+)
+
+type CustomMetadata map[string]string
+
+const (
+	maxKeys               = 64
```

Comment:
Are these universally applied to all custom metadata of any type? Is there any exceptions that need to be noted perhaps in a comment?

Reply:
That's a great question. There are some comments in the KVv2 implementation I think that I'll bring over and modify if necessary. Thanks for the input!